src/main/java/seedu/address/logic/commands/FindPersonCommand.java

```diff
@@ -8,15 +8,15 @@
 import seedu.address.model.person.NameStartsWithKeywordPredicate;
 
 /**
- * Finds and lists all people in Plannit whose name starts with the argument keyword.
+ * Finds and lists all person in Plannit whose name starts with the argument keyword.
```

Comment:
```suggestion
 * Finds and lists all persons in Plannit whose name starts with the argument keyword.
```

Another grammatical error?